fw/tls_conf.c

```diff
@@ -201,6 +202,18 @@ tfw_tls_set_cert(TfwVhost *vhost, TfwCfgSpec *cs, TfwCfgEntry *ce)
 		goto err;
 	}
 
+	if ((flags = ttls_x509_check_cert_validity(&conf->crt))) {
```

Comment:
Could you please add a comment why we check only dates, and don't do the full check like with `ttls_x509_crt_verify_with_profile()`, which checks also algorithms and so on.